.github/workflows/e2e-manual.yaml

```diff
@@ -0,0 +1,94 @@
+name: Manual E2E
+on:
+  # when https://github.com/community/community/discussions/11795 is resolved
+  # we will be able to dynamically build up the list of valid inputs.
+  # for now this needs to be manual.
+  workflow_dispatch:
+    inputs:
+      test-entry-point:
+        description: 'Test entry point'
+        required: true
+        type: choice
+        options:
+          - TestFeeMiddlewareTestSuite
+      chain-a-image:
+        description: 'The image to use for chain A'
+        required: true
+        type: string
+        default: "ghcr.io/cosmos/ibc-go-simd-e2e"
+      chain-a-tag:
+        description: 'The tag to use for chain A'
+        required: true
+        type: choice
+        default: main
+        options:
+          - main
+          - v4.0.0-rc3
+          - v3.0.0
+          - v2.2.0
+          - v2.1.0
+          - v2.0.0
```

Comment:
Are we missing `v3.1.0` here?

Reply:
we are, but that image hasn't actually been built yet. That will have to come in a follow up!